Lib/asyncio/base_events.py

```diff
@@ -168,6 +168,17 @@ def _run_until_complete_cb(fut):
     futures._get_loop(fut).stop()
 
 
+if hasattr(socket, 'TCP_NODELAY'):
```

Comment:
I'd add a new `asyncio.sockutils` module for this kind of stuff.

Reply:
`sockutils.py` sounds great but in the case I doubt if the PR can be backported.

Reply:
Indeed, I suggest to do that in a follow up PR specifically for the master branch.

Reply:
Created https://bugs.python.org/issue35392 for it.